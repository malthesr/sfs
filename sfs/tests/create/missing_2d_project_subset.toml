args = [
  "create",
  "-vv",
  "-P",
  "1,1",
  "-s",
  "sample0=A,sample1=B,sample2=A,sample4=B",
  "tests/create/missing.bcf"
]
