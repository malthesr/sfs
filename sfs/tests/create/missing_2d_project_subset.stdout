#SHAPE=<3/3>
1.000000 1.000000 1.500000 0.500000 0.500000 0.000000 0.500000 0.000000 0.000000
